src/Steppenwolf/Pages/Admin/BlogPostModel.cs

```diff
@@ -15,5 +17,10 @@ public class BlogPostModel
         [Required]
         [StringLength(300, MinimumLength = 150)]
         public string Preview { get; set; }
+
+        [Required]
+        [MinLength(1)]
+        [MaxLength(5)]
```

Comment:
Suggestion for limiting categories in the header - take top 5 with most posts and return them. We could include in the future in a search the rest of the categories.